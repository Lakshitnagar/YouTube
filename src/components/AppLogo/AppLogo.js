import React from 'react';
import logo from '../../assets/images/logo.svg';
import './AppLogo.scss';

function AppLogo() {
    return (
        <div className="appLogo">
            <img src={logo} alt={'logo'} height={30}/>
        </div>
    );
}

export default AppLogo;
