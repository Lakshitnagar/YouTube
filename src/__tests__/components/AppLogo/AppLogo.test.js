import React from 'react';
import {shallow} from "enzyme";
import {shallowToJson} from "enzyme-to-json";

import AppLogo from "../../../components/AppLogo/AppLogo";

describe('AppLogo', function () {
    it('should render and match snapshot', function () {
        const component = <AppLogo />;

        const wrapper = shallow(component);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
