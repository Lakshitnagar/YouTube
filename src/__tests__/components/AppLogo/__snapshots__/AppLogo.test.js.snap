// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLogo should render and match snapshot 1`] = `
<div
  className="appLogo"
>
  <img
    alt="logo"
    height={30}
    src="logo.svg"
  />
</div>
`;
